docs/sources/alerting/monitor/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/monitor/
+description: Monitor alerting metrics and data
+keywords:
+  - grafana
+  - alert
+  - monitoring
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Monitor
+title: Monitor alerting metrics
```

Comment:
Do you also want to mention insights logs?

Reply:
i think i'll leave it out now - also have left out alerting insights until that's merged - and until i split it into cloud only docs